src/test/scala/com/cognite/sdk/scala/v1/TimeSeriesTest.scala

```diff
@@ -104,24 +105,42 @@ class TimeSeriesTest extends SdkTest with ReadBehaviours with WritableBehaviors
     fewTimeSeriesForOtherAsset.size should be (1)
 
     val timeSeriesForBothAssets = client.timeSeries
-      .filter(
+      .filterPartitions(
```

Comment:
I wonder if we should have one of them still use `filter`, or have `filter` in addition to `filterPartitions`? A test that they contain the same items could be nice.

Reply:
Great idea! Should have thought of that, added.